apps/api/tsconfig.json

```diff
@@ -5,6 +5,12 @@
     "declarationMap": false,
     "outDir": "./dist",
     "rootDir": ".",
+    "baseUrl": ".",
+    "paths": {
+      "@/*": ["./src/*"],
+      "@/scripts/*": ["./scripts/*"],
+      "@/e2e/*": ["./e2e/*"]
+    },
```

Comment:
Made `src` the primary/short/easy `@/...` path with `scripts` and `e2e` with their own paths.